lib/miq_automation_engine/models/miq_ae_domain.rb

```diff
@@ -45,6 +53,45 @@ def available_version
     version_field.try(:fetch_path, 'field', 'default_value')
   end
 
+  def self.import_git_repo(domain_name, git_repo, ref = DEFAULT_BRANCH, ref_type = BRANCH)
+    MiqAeDomain.find_by_name(domain_name).try(:destroy)
```

Comment:
We are trying to move away from `find_by_name`...prefer `find_by(:name => domain_name)`